AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -65,6 +67,17 @@
  */
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
+/**
+ * @abstract Return the calculated layout.
+ *
+ * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
```

Comment:
Sorry I slept before your latest message arrived. I can help you privately. Feel free to send me an email. My email address is huy.nguyen151190 at gmail dot com.

On 6 Jul 2015 at 10:44:30 pm, markmark1 (notifications@github.com) wrote:

In AsyncDisplayKit/ASDisplayNode+Subclasses.h:

> @@ -65,6 +67,17 @@
>   */
>  @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
>   
> +/**
> + * @abstract Return the calculated layout.
> + *
> + * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
Can you give a sample as our asdk and non asdk project have  lots of clips to bounds rounded corners
…
—
Reply to this email directly or view it on GitHub.